core/packages/test/scripts/start-ethereum.sh

```diff
@@ -8,6 +8,7 @@ start_chains()
 {   #1 start ethereum and deploy contracts
     echo "start ethereum and deploy bridge contrancts"
     deploy_ethereum
+    echo "Chain started!"
```

Comment:
Nit: Shouldn't it maybe say:
```suggestion
    echo "Ethereum started"
```